content/greenlake/data-services-on-the-hpe-greenlake-platform/home.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Data Services on the HPE GreenLake platform
+version: 0.1.1
+description: The landing page for all Storage BU related SaaS and cloud services
+  with future unified API in the plan
+image: /img/greenlake/dscc-icon-transparent.png
+width: large
+priority: 1
+active: false
+tags:
+  - hpe-greenlake
+  - api
+  - data-services-for-hpe-greenlake
+---
+<style>
```

Comment:
Hi Ron,

In the preview https://deploy-preview-2531--hpe-dev-portal.netlify.app/greenlake/data-services-on-the-hpe-greenlake-platform/home/ , the font for the numbered item lists is different than the rest of the content. 

Can you please try to insert the following style statement at the top of the page instead of the other two style statement to see whether the rendering would be better? 

Note: The fonts may appear bigger than the standard text in the right panel of the CMS editor. This is expected. However, the Netlify process should render the text correctly (as standard text size) when creating the “Preview” of the post and when publishing the post to the HPE Developer Community portal.


<style>
li {
   font-size: 27px;
   line-height: 33px;
   max-width: none;
}
</style>

 instead of the two styles:

<style>
ul li{
 font-size:27px;
}
</style>

<style>
ol li{
 font-size:27px;
}
</style>

Reply:
You should also add in the same style section:    line-height: 33px;
To give a little more room between lines.

Reply:
Got it.. I have applied the single style as suggested by Denis which already included Didier suggestion of the line-height: 33px